code/src/Components/InputTypeRadio.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const InputTypeRadio = ({className, header, array, selected, setSelected}) => {
+    return(
+        <div className="favourite-color-container">
+        <h3 tabIndex="0">{header}</h3>        
```

Comment:
I think it might be because <h3> is inside a div and when landing on this section the screen-reader is already inside that div.
If I tab through the hole page it will read <h3>.
I could remove the div-container i guess but it will mess up the styling so I will leave it for now.